extension/secrets.go

```diff
@@ -25,7 +25,9 @@ func LoadSecrets() {
 	fmt.Println("Current timestamp:", time.Now().UnixMilli())
 
 	secretIds := readSecretIdsFromEnvironmentWhenStartsWithSecret()
-	secrets := getSecretValuesFromListOfSecretIds(secretIds)
+	secretsChannel := make(chan Secret, len(secretIds))
+	go getSecretValuesFromListOfSecretIds(secretIds, secretsChannel)
```

Comment:
I don't think the "go" command is correct here? As far as I understood it the go is the fan out. I think what happens here is that the get secretValues function is "async" but the secret values are fetched in sequence ?

Reply:
Im not sure either... could be. At least it felt like it was still worked in sequence. Also im not sure if it now works allright 😅